pom.xml

```diff
@@ -44,66 +44,49 @@
     <url>https://github.com/wildfly/wildfly-bom-builder-plugin</url>
   </scm>
 
-
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <mavenVersion>3.2.1</mavenVersion>
-    <mavenPluginPluginVersion>3.2</mavenPluginPluginVersion>
+    <version.mavenPluginTools>3.8.2</version.mavenPluginTools>
+    <version.wildFlyChannels>1.0.0.Final</version.wildFlyChannels>
   </properties>
 
-  <dependencies>
-
-    <dependency>
-      <groupId>org.apache.maven</groupId>
-      <artifactId>maven-plugin-api</artifactId>
-      <version>3.2.1</version>
-    </dependency>
-
-    <dependency>
-      <groupId>org.apache.maven</groupId>
-      <artifactId>maven-artifact</artifactId>
-      <version>${mavenVersion}</version>
-    </dependency>
-
-    <dependency>
-      <groupId>org.apache.maven</groupId>
-      <artifactId>maven-core</artifactId>
-      <version>${mavenVersion}</version>
-    </dependency>
-
-    <dependency>
-      <groupId>org.apache.maven</groupId>
-      <artifactId>maven-model-builder</artifactId>
-      <version>${mavenVersion}</version>
-    </dependency>
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.apache.maven.plugin-tools</groupId>
+        <artifactId>maven-plugin-tools-annotations</artifactId>
+        <version>${version.mavenPluginTools}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.wildfly.channel</groupId>
+        <artifactId>maven-resolver</artifactId>
+        <version>1.0.0.Final</version>
```

Comment:
should this be `${version.wildFlyChannels}` ?

Reply:
yep, thanks for fixing that too